adds/start.sh

```diff
@@ -54,11 +54,21 @@ configure_homeserver_yaml() {
 	mv ${ymltemp} "${filepath}"
 }
 
+configure_log_config() {
+        sed -i "s|.*filename:\s/homeserver.log|    filename: /data/homeserver.log|g" "/data/${SERVER_NAME}.log.config"
+}
+
```

Comment:
there is a problem that if this is not set... it will write the logfile to the container root. is there a better way to set it?